site/content/docs/5.0/getting-started/introduction.md

```diff
@@ -120,7 +120,7 @@ Bootstrap requires the use of the HTML5 doctype. Without it, you'll see some fun
 
 ### Responsive meta tag
 
-Bootstrap is developed *mobile first*, a strategy in which we optimize code for mobile devices first and then scale up components as necessary using CSS media queries. To ensure proper rendering and touch zooming for all devices, **add the responsive viewport meta tag** to your `<head>`.
+Bootstrap is developed *mobile-first*, a strategy in which we optimize code for mobile devices first and then scale up components as necessary using CSS media queries. To ensure proper rendering and touch zooming for all devices, **add the responsive viewport meta tag** to your `<head>`.
```

Comment:
in this case, both with and without the hyphen works, I think - and there's quite a few instances of it being used without. Slight preference for having it without, here, as it's not directly an adjective (i.e. if the sentence was "is developed with a mobile-first approach" then yes, i'd go with hyphen ... but in this case it looks better to me without)